web-app/tests/README.md

```diff
@@ -1,52 +1,64 @@
-This folder container playwright, e2e tests for web-app,
+# Web App Test Harness
+
+This folder contains playwright, e2e tests for web-app,
 running against a local or remote backend in proxy mode.
 
+## Bring up the platform behind local (vite dev server) proxy
 
-## Bring up backend behind local (vite dev server) proxy
+Bring up test backend services (identity provider, database, etc.):
 
-To configure backend, you can either use the opentdf quickstart
-or the backend repository directly.
+```sh
+cd .github/workflows/roundtrip/
+docker compose up -d
+```
 
-### Quickstart
+> You can leave off the `-d` to track the logs; 
+> if you do this, open a new terminal
+> and continue with the instructions below.
 
-First, connect to your cluster or spin up a local cluster (e.g. using kind or minikube).
+Configure the identity provider with some test users.
+Please remain in the `roundtrip` test directory.
 
-Next, check out the opentdf repository and start the quickstart.
-```
-git clone https://github.com/opentdf/opentdf.git
-cd opentdf/quickstart
-export OPENTDF_INGRESS_HOST_PORT=5432
-export OPENTDF_LOAD_FRONTEND=
-tilt up
+```sh
+go run github.com/opentdf/platform/service@latest provision keycloak
+./config-demo-idp.sh
 ```
 
-Finally, bring up the web app:
+If you haven't already done so, initialize your service keys.
+Be careful!
+If you lose these keys you will lose access to all TDFs encrypted with them.
```

Comment:
These keys are stored in the directory `<where>`

Reply:
For now they are just created whereever you put them. They are loaded by o/platform/service from the current working directory.